cmake/FindSCALAPACK.cmake

```diff
@@ -0,0 +1,39 @@
+#
+# Distributed Linear Algebra with Future (DLAF)
+#
+# Copyright (c) 2018-2023, ETH Zurich
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+include(FindPackageHandleStandardArgs)
+find_package(PkgConfig REQUIRED)
+
+pkg_search_module(_SCALAPACK scalapack)
+
+find_library(
+  SCALAPACK_LIBRARIES NAME scalapack
+  HINTS ${_SCALAPACK_LIBRARY_DIRS}
+        ENV
+        SCALAPACKROOT
+        SCALAPACK_ROOT
+        SCALAPACK_PREFIX
+        SCALAPACK_DIR
+        SCALAPACKDIR
+        /usr
+  PATH_SUFFIXES lib
+)
+
+find_package_handle_standard_args(SCALAPACK DEFAULT_MSG SCALAPACK_LIBRARIES)
+
+mark_as_advanced(SCALAPACK_LIBRARY)
+
+if(SCALAPACK_FOUND)
+  if(NOT TARGET DLAF::SCALAPACK)
+    add_library(DLAF::SCALAPACK INTERFACE IMPORTED GLOBAL)
+  endif()
+
+  target_link_libraries(DLAF::SCALAPACK INTERFACE "${SCALAPACK_LIBRARY}")
+endif()
```

Comment:
Here there is a mixup of `_LIBRARY` and `_LIBRARIES`. I commented also in the other file, but actually it's up to you deciding the final name for the variable 😄